### ISSUE TITLE
SA1024 conflicting with SA1003

### ISSUE THREAD

**Issue by sleemer** [DotNetAnalyzers/StyleCopAnalyzers#3073]
In code like this:
```csharp
Console.WriteLine($"{_nextId++ :d10}")
```
I get conflict of  SA1024 (Colon should not be preceded by a space) with SA1003 (Operator '++' should be followed by whitespace)

I would suggest using SA1003 over SA1024 (as in the code above).

**Comment by sharwell** [comment]
SA1024 should have precedence in this case.

**Comment by sharwell** [comment]
Actually it looks like both diagnostics are reported in similar scenarios (e.g. ternary operators). I'll update it to not require a space here.
